ext/native-decls/RegisterCommand.md

```diff
@@ -19,24 +19,31 @@ The restricted bool is not used on the client side. Permissions can only be chec
 ![](https://i.imgur.com/TaCnG09.png)
 
 
-## Parameters
+## Required Arguments
 * **commandName**: The command you want to register.
 * **handler**: A handler function that gets called whenever the command is executed.
+
+## Optional Arguments
 * **restricted**: If this is a server command and you set this to true, then players will need the command.yourCommandName ace permission to execute this command.
 
+## Handler function parameters
+* **playerSource**: The player font which executed the command.
+* **args**: returns the string the player entered after the command.
```

Comment:
it doesn't return a string, it returns a string array of arguments

Reply:
Okay, could you make that change? It's because I never played Fivem, I just study and suggest something in the documentation.

Reply:
I put String because the page says nothing about returning an array.

Reply:
https://docs.fivem.net/docs/scripting-manual/migrating-from-deprecated/creating-commands/

Reply:
I would love for you to add: Handler function parameters

Reply:
I would like to keep this edition of mine, but instead of 'playerSource' leave 'source': D

Reply:
that's it, thank you all!